src/quo2/components/settings/data_item/content/loading.cljs

```diff
@@ -0,0 +1,7 @@
+(ns quo2.components.settings.data-item.content.loading
+  (:require [react-native.core :as rn]
+            [quo2.components.settings.data-item.style :as style]))
+
+(defn view
+  [{:keys [size blur? theme]}]
+  [rn/view {:style (style/loading-container size blur? theme)}])
```

Comment:
We don't need a namespace for this part of a component. Just create it as other function in the main component namespace

Reply:
Done